tests/playwright/sanity/assets/dev/js/editor/elements/views/widget.test.js

```diff
@@ -0,0 +1,16 @@
+const { test, expect } = require( '@playwright/test' );
```

Comment:
@hein-obox I think the filename needs to be more descriptive as well. You will see there is a file `hidden-widgets-on-search.test.js`. So maybe call yours `widget-empty-state.test.js`

Reply:
For short descriptions of test modules there is the `test.describe()` method

Reply:
@udidol @louiswol94 

Can we do a quick huddle to discuss this? That might help for future reference as well. 

@udidol I have noticed that in most tests the describe method is used to indicate the run command name?
@louiswol94 I have folllowed the file and folder structure from the exisiting JS files. If we follow that method, then I am not sure if I should change the filename.

Reply:
It's the run command name and a very short general description for a group of tests

Reply:
I think this is ok